### PR TITLE
Record node id in perflog

### LINKED ISSUE #248: Add node id(s) to perflog
- [ ] Add line to ReFrame config
- [ ] Add logic to postprocessing to parse the data

### PR DESCRIPTION
**Pull request by tkoskela**
Closes #248

### CODE REVIEW

**Comment by tkoskela** [comment]
Post-processing tests have failed on my change :sob: Do I need to add the node id to `EXCPECTED_FIELDS`? @ilectra

**Comment by ilectra** [comment]
> Post-processing tests have failed on my change 😭 Do I need to add the node id to `EXCPECTED_FIELDS`? @ilectra

Yes

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ on:
       - '.github/workflows/ci.yml'
       - 'MANIFEST.in'
       - 'pyproject.toml'
+      - 'reframe_config.py'
```
Comment (ilectra) [inline .github/workflows/ci.yml:11]: Don't you need the whole path `benchmarks/reframe_config.py` here?

Comment (tkoskela) [reply .github/workflows/ci.yml:11]: Hmm, I would think so, but it triggered without. Confusing

Ah, it triggered because the workflow itself changed

**Review by ilectra — APPROVED** [review]

**Pull request merged**